development/tests/test_match_details.py

```diff
@@ -83,10 +86,14 @@ def test_should_shows_matches_if_connected_user_has_matches_played(
 
     @patch('development.views.get_logs')
     @patch('development.views.generate_text')
+    @patch.object(FilterLogs, 'possible_states', new_callable=PropertyMock)
+    @patch.object(FilterLogs, 'possible_actions', new_callable=PropertyMock)
     def test_should_shows_pass_data_to_template_when_it_is_received_from_server(
         self,
-        mocked_get_log,
+        mokced_possible_action,
+        mokced_possible_states,
```

Comment:
Replace `mokced` with `mocked`

Reply:
haha sorry, always make typos. Thank you

Reply:
Done